react/__tests__/components/BuyButton.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import BuyButton from '../../BuyButton'
+
+describe('<BuyButton />', () => {
+  let wrapper
+
+  const renderComponent = customProps => {
+    const props = {
+      ...customProps,
+      showToast: () => {},
+    }
+
+    const component = shallow(<BuyButton {...props}> Test </BuyButton>)
+
+    return {
+      component,
+    }
+  }
+
+  beforeEach(() => {})
+
+  it('should be rendered', () => {
+    wrapper = renderComponent()
```

Comment:
```suggestion
    const wrapper = renderComponent()
```
Let's avoid possible colateral effects of using a global variable. The same for the other `it`s